### ISSUE TITLE
Auto import passwords from other browsers

### ISSUE THREAD

**Issue by jasonLaster** [replayio/gecko-dev#323]
Idea for this one: in addition to/instead of supporting password managers, maybe you can export cookies from Chrome/Firefox to Replay

I know that's what Mighty did for their browser. Not sure if they posted their solution: https://twitter.com/Suhail/status/1308583172829896704?s=20

Suhail (@Suhail)
> If you’ve ever exported chrome cookies without a browser extension, lmk! We are trying to figure out the easiest way to > auto-login users into websites while using Mighty since the cookies don’t carry over across device.

I'll echo the frustration when browsers/apps don't have my password manager --> I have to go to chrome, click lastpass, search for my username, click copy password, go back to replay (or the other app) and paste it in---it would probably feel magical to see a popup that asks whether I want to import my browser sessions into replay and have it work

**Comment by jasonLaster** [comment]
<img width="552" alt="Screen Shot 2021-06-09 at 6 29 44 PM" src="https://user-images.githubusercontent.com/254562/121450497-bbc8d580-c950-11eb-8105-08aedbb95a5e.png">
<img width="552" alt="Screen Shot 2021-06-09 at 6 26 40 PM" src="https://user-images.githubusercontent.com/254562/121450501-be2b2f80-c950-11eb-92c7-04f1a79eda3d.png">
<img width="552" alt="Screen Shot 2021-06-09 at 6 20 14 PM" src="https://user-images.githubusercontent.com/254562/121450502-bec3c600-c950-11eb-8b18-92bfcc085b55.png">

**Comment by jasonLaster** [comment]
Here's the JS for the migration wizard
https://searchfox.org/mozilla-central/source/browser/components/migration/content/migration.js#35

**Comment by jasonLaster** [comment]
We should prioritize adding an import settings step to the welcome flow.

1. The welcome tab next button should direct folks to an Import Settings page. It's that important
2. The page should have a prominent import button and less prominent skip button
3. clicking the button should send a WebChannel message to DevToolsStartup where we can open the wizard
4. From DevToolsStartup, we should be able to use `MigrationUtils` to launch the wizard

```js
export function setUserInBrowserPrefs(user: { sub: string } | null) {
  const _user = user === null ? "" : user;
  window.dispatchEvent(
    new window.CustomEvent("WebChannelMessageToChrome", {
      detail: JSON.stringify({
        id: "record-replay",
        message: { import: true },
      }),
    })
  );
}

```

```js
const { MigrationUtils } = ChromeUtils.import("resource:///modules/MigrationUtils.jsm")
```

**Comment by ryanjduffy** [comment]
Fixed by RecordReplay/website#117 and RecordReplay/gecko-dev#421
